scio-bigquery/src/test/scala/com/spotify/scio/bigquery/types/SchemaProviderTest.scala

```diff
@@ -77,4 +77,32 @@ class SchemaProviderTest extends FlatSpec with Matchers {
     SchemaProvider.schemaOf[RepeatedNested] should equal (parseSchema(recordFields("REPEATED")))
   }
 
+  case class User(@description("user name") name: String, @description("user age") age: Int)
+  case class Account(@description("account user") user: User,
+                     @description("in USD") balance: Double)
```

Comment:
Yeah but no-desc cases are covered above. Don't wanna over do it and handcraft exhaustive test cases.